Tools/LyTestTools/ly_test_tools/o3de/multi_test_framework.py

```diff
@@ -1201,8 +1205,11 @@ def _exec_multitest(self,
                 # If it didn't then it will have "Unknown" as the type of result.
                 results = self._get_results_using_output(test_spec_list, output, executable_log_content)
                 if not len(results) == len(test_spec_list):
-                    raise EditorToolsFrameworkException("bug in get_results_using_output(), the number of results "
-                                                        "don't match the tests ran")
+                    logger.debug("\n=============Debug===========\n"
```

Comment:
IMO a debug log shouldn't need big headings like this, but I'll have to see it in action to know whether it's useful or frustrating.

Reply:
removed header